packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.base.tsx

```diff
@@ -29,6 +31,16 @@ export interface IResizeGroupState {
    * to find a transformation of the data that makes everything fit.
    */
   resizeDirection?: 'grow' | 'shrink';
+
+  /**
+   * When we get new props, we typically first create a measure div and in the next render put this data into a new rendered div
+   * (See the render function below)
+   *
+   * When we do this, React unmounts the measured div and its children fully and remounts all of them in the rendered div.
+   * To avoid this, we flip the keys to "fool" React so that it updates the previously measure div to be the render div
+   * instead of remounting.
+   */
+  measureDivKey?: ResizeGroupDivKey;
```

Comment:
That's pretty interesting. :)

Reply:
This is only for the initial render that we do this key swap right? If not I fear we will end up doing weird things like losing focus with this trick

Reply:
nevermind, it looks like it is for every render. I'm guessing the Ribbon would lose focus if it had it in this case?

Reply:
Yeah I think this might lead to focus bugs. Did not verify it but makes sense.

Reply:
I can this would lead enough inconsistency that I think the perf. gains might not be worth it. 
Would it even make sense to handhold focus when we do this switching ? Like keep track of the previousFocusedElement inside the ResizeGroup's contents and see it still is present after the new contents are rendered. It will however raise superfluous "blur" and "focus" event on that element.